<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">

        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>463</width>
                <height>400</height>
            </rect>
        </property>

        <property name="windowTitle">
            <string>Dialog</string>
        </property>

        <widget class="QLabel" name="label_7">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>20</y>
                    <width>581</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>SELECT AN OBJECT</string>
            </property>
        </widget>

        <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>0</y>
                    <width>350</width>
                    <height>370</height>
                </rect>
            </property>

            <layout class="QGridLayout" name="gridLayout">

                <!-- row 0 -->
                <item row="0" column="0">
                    <layout class="QVBoxLayout" name="verticalLayout_2">x
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_4">
                                <item>
                                    <widget class="QLabel" name="label_0">
                                        <property name="text">
                                            <string>Current Object:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLabel" name="center_objs">
                                        <property name="text">
                                            <string>None</string>
                                        </property>
                                        <property name="wordWrap">
                                            <bool>true</bool>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </item>

                <!-- row 1 -->
                <item row="1" column="0" colspan="2">
                    <widget class="QFrame">
                        <property name="frameShape">
                            <enum>QFrame::HLine</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                    </widget>
                </item>

                <!-- row 2 -->
                <item row="2" column="0">
                    <widget class="QLabel" name="label_1">
                        <property name="text">
                            <string>Distort:</string>
                        </property>
                    </widget>
                </item>
                <item row="2" column="1">
                    <layout class="QGridLayout" name="gridLayout_4" >
                        <item row="0" column="0">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>X: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QDoubleSpinBox" name="X_input"/>
                        </item>

                        <item row="0" column="2">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>Y: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="3">
                            <widget class="QDoubleSpinBox" name="Y_input"/>
                        </item>

                        <item row="0" column="4">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>Z: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="5">
                            <widget class="QDoubleSpinBox" name="Z_input"/>
                        </item>
                    </layout>
                </item>

                <!-- row 3 -->
                <item row="3" column="0" colspan="2">
                    <widget class="QFrame">
                        <property name="frameShape">
                            <enum>QFrame::HLine</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                    </widget>
                </item>

                <!-- row 4 -->
                <item row="4" column="0">
                    <widget class="QLabel" name="label_1">
                        <property name="text">
                            <string>Distortions:</string>
                        </property>
                    </widget>
                </item>
                <item row="4" column="1">
                    <layout class="QGridLayout" name="gridLayout_4" >
                        <item row="0" column="0">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>Count: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QSpinBox" name="count_input"/>
                        </item>

                        <item row="1" column="0">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>X Min: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QDoubleSpinBox" name="X_min_input"/>
                        </item>
                        <item row="1" column="2">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>X Max: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="3">
                            <widget class="QDoubleSpinBox" name="X_max_input"/>
                        </item>

                        <item row="2" column="0">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>Y Min: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QDoubleSpinBox" name="Y_min_input"/>
                        </item>
                        <item row="2" column="2">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>Y Max: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="3">
                            <widget class="QDoubleSpinBox" name="Y_max_input"/>
                        </item>

                        <item row="3" column="0">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>Z Min: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="1">
                            <widget class="QDoubleSpinBox" name="Z_min_input"/>
                        </item>
                        <item row="3" column="2">
                            <widget class="QLabel">
                                <property name="text">
                                    <string>Z Max: </string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="3">
                            <widget class="QDoubleSpinBox" name="Z_max_input"/>
                        </item>

                    </layout>
                </item>

                 <!-- row 12 -->
                <item row="12" column="0">
                    <widget class="QLabel" name="warnings">
                        <property name="text">
                            <string/>
                        </property>
                    </widget>
                </item>

            </layout>
        </widget>

        <widget class="QPushButton" name="apply_button">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>280</y>
                    <width>100</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="text">
                <string>Apply</string>
            </property>
        </widget>

        <widget class="QPushButton" name="close_button">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>320</y>
                    <width>100</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="text">
                <string>Close</string>
            </property>
        </widget>

    </widget>

    <resources/>
    <connections/>
</ui>
